couchdb/src/test/java/org/hibernate/ogm/datastore/couchdb/test/crud/EntityWithByteArrayTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.couchdb.test.crud;
+
+import java.util.Arrays;
+import java.util.Random;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.ogm.utils.OgmTestCase;
+import org.hibernate.ogm.utils.TestForIssue;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * @author Hardy Ferentschik
+ */
+public class EntityWithByteArrayTest extends OgmTestCase {
```

Comment:
> Just generalizing a user created specific issue seems wrong to me.
> 
> It's what we often do. Naturally people notice certain shortcomings only on the store(s) they use. IMO it makes sense to handle and fix it generically if we find out it applies to all stores actually.

I don't think we can ask the user to install all the supported datastores and check if the problem happens somewhere else before opening a JIRA, It's up to us to recognize the features that must be consistent across the datastores.

> Which are both minimal.

But it's unecessary, and it can be easly avoided with something that we already have.